MkDocs/hpu_8.6/index.md

```diff
@@ -0,0 +1,89 @@
+![img](/img/hpu_8.6/banner.png){:style="height:150px;width:800px"}
+
+# Welcome to the Maximo Application - Health and Predict for Utilities Overview Lab <br> (Version: 8.6)
+
+You will learn a high level overview of the Maximo Application Suite applications: Monitor, Digital Twin Exchange, Visual Inspection, Manage, Mobile, Assist, Parts Identifier, Safety, Predict, and Health with a story of a waterwaste plant. You will understand how all of these applications work together to run operations within a business. 
+
+Students doing the demo in this lab should have been provided access to Maximo Application Suite v8.5 instance by the Lab Facilitator or your IBM Rep.  
+
+Preparing for delivering the demo should read the Pre-Requisites located in the left column where the indexes for the entire lab are.
+
+
+
+![img](/img/hpu_8.6/mkdocstab.png){:style="height:400px;width:400px"}
+
+See [Contributing to Labs](../../about) if you are interested in providing feedback by submitting enhancements or bugs.  We also 
+welcome your contributions to existing or new exercises. 
+
+## Story
+
+A Water Resource Authority needs to manage water and wastewater treatment assets across multiple cities and regions from a single application.  Remote operational support engineers need a single dashboard to manage sites and assets.  The Technician needs to ensure his safety and have access to accurate details about the work he is assigned to perform. The Asset Analyst and Asset Manager tap into digital twins to improve asset management operations. Safety Officers are ensuring appropriate safety polices in place to minimize risk and provide safer working environments for the technicians.  The reliability engineer needs to understand the asset health and prevent predicted failures by properly servicing the rotating assets like centrifugal pumps.
+
+## Personas
+
+•	Remote Operational Support<br>
+•	Asset Analyst<br>
+•   Asset Manager<br>
+•   Maintenance Supervisor<br>
+•   Technician<br>
+•   Safety Officer<br>
+•   Reliability Engineer<br>
```

Comment:
Please remove all the <br> markdown files you added.  You shouldn't have to include the HTML in the markdown files to get a carriage return.  It also makes the markdown more complicated and harder to read.  it also increases the size of the site making deployments take longer.